openwpm/utilities/platform_utils.py

```diff
@@ -125,7 +125,7 @@ def get_configuration_string(manager_params, browser_params, versions):
         # Separate out long profile directory strings
         profile_dirs[browser_id] = str(item.pop("seed_tar"))
         archive_dirs[browser_id] = str(item.pop("profile_archive_dir"))
-        js_config[browser_id] = item.pop("js_instrument_settings")
+        js_config[browser_id] = item.pop("cleaned_js_instrument_settings")
```

Comment:
Should we pop off both?

Reply:
What do you mean by both?

Reply:
Now we only pop off the expanded version, however if a user were to supply their own expanded config, that would still ruin our printing.

Reply:
I see. If both are present than it makes sense to remove both from printing.